frontier_generators.gemspec

```diff
@@ -1,18 +1,18 @@
+  s.homepage = "https://github.com/thefrontiergroup/frontier_generators"
+  s.rubygems_version = "2.4.6"
+  s.summary = "Comprehensive generators for CRUD"
```

Comment:
Damn... I actually didn't change this... will revert this guy, don't know why these changes are in here....